osx/make-installer.sh

```diff
@@ -3,51 +3,51 @@
 # abort script if any command fails
 set -e
 
-# Retrieve latest source code, keep history as it's useful to edit sometimes
-if [ ! -d "source" ]; then
-  git clone https://github.com/Mudlet/Mudlet.git source
+# set path to find macdeployqt
+PATH=/usr/local/opt/qt5/bin:$PATH
+
+cd source/build
+
+# install installer dependencies
+brew update
+BREWS="sqlite3 lua@5.1 node wget"
+for i in $BREWS; do
+  brew outdated | grep -q $i && brew upgrade $i
+done
+for i in $BREWS; do
+  brew list | grep -q $i || brew install $i
+done
+if [ ! -f "macdeployqtfix.py" ]; then
+  wget https://raw.githubusercontent.com/aurelien-rainone/macdeployqtfix/master/macdeployqtfix.py
 fi
+echo "Need root permissions to install lua rocks..."
+sudo luarocks-5.1 install LuaFileSystem
```

Comment:
Could you move the luarocks install to be at the top, so the script asks you for the sudo password once at the start and then you can look away?